errors.go

```diff
@@ -54,6 +54,11 @@ var (
 	// ErrTooManyRowGroups is returned when attempting to generate a parquet
 	// file with more than MaxRowGroups row groups.
 	ErrTooManyRowGroups = errors.New("the limit of 32767 row groups has been reached")
+
+	// ErrConversion is used to indicate that a conversion betwen two values
+	// cannot be done because there are no rules to translate between their
+	// physical types.
+	ErrInvalidConversion = errors.New("invalid conversion between parquet values")
```

Comment:
If you are hitting this error you would likely want information about what types you are trying to convert between

Ie. I imagine a much more useful error would be 

cannot convert between %q and %q (no rules exist to perform the conversion)

Reply:
This is just the sentinel value to test with `errors.Is`, we always wrap the error where we use it with extra context like what you suggested.